### PR TITLE
To make modular for files install.sh/upgrade.sh for Centos, RHEL,Oracle for 7.x series

### PR DESCRIPTION
**Pull request by DivyaJyothiKasam**
Created tar file:
lis-rpm-buildfunction-2018-12-04.14.25.tar.gz

Below are the logs:
[root@centos74x64 ~]# yum list installed | grep micro
Repodata is over 2 weeks old. Install yum-cron? Or run: yum makecache fast
kmod-microsoft-hyper-v.x86_64        buildfunction-20181204      installed
microcode_ctl.x86_64                 2:2.1-22.5.el7_4            @updates
microsoft-hyper-v.x86_64             buildfunction-20181204      installed
microsoft-hyper-v-debuginfo.x86_64   buildfunction-20181204      installed
[root@centos74x64 ~]# modinfo hv_vmbus
filename:       /lib/modules/3.10.0-862.2.3.el7.x86_64/weak-updates/microsoft-hyper-v/hv_vmbus.ko
version:        buildfunction
license:        GPL
retpoline:      Y
rhelversion:    7.5
srcversion:     581BFCB32D09E7A1B019591
alias:          acpi*:VMBus:*
alias:          acpi*:VMBUS:*
depends:
vermagic:       3.10.0-862.el7.x86_64 SMP mod_unload modversions
parm:           affinity_mode:vmbus channel cpu affinity mode: 0, 1 (int)

### CODE REVIEW

File: LISISO/CentOS73/install.sh [hunk 1]
```diff
@@ -6,437 +6,69 @@
...
-regex13='3.10.0-514.48.5'
-regex14='3.10.0-514.53.1'
-regex15='3.10.0-514.55.4'
-regex16='3.10.0-514.58.1'
-regex17='3.10.0-514.61.1'
-
-if [[ "$kernelver" =~ $regex ]]; then
-   {
-        cd update
-        kmodrpm=`ls kmod-microsoft-hyper-v-*.x86_64.rpm`
```
Comment (prashantmhetar) [inline LISISO/CentOS73/install.sh]: Please make a common function for this code

File: LISISO/CentOS73/install.sh [hunk 2]
```diff
@@ -6,437 +6,69 @@
...
+{
+	i=$((i+1))
+	regex =$1
+	kernelver=`uname -r`
+	if [[ "$kernelver" =~ $regex ]]   ;
+	then
+		cd "update${i}"
+		kmodrpm=`ls kmod-microsoft-hyper-v-*.x86_64.rpm`
+		msrpm=`ls microsoft-hyper-v-*.x86_64.rpm`
+	        if [ "$kmodrpm" != "" ] && [ "$msrpm" != ""  ]; then
```
Comment (prashantmhetar) [inline LISISO/CentOS73/install.sh]: Below else case uses same code..please make a common function

File: LISISO/CentOS73/upgrade.sh [hunk 3]
```diff
@@ -6,439 +6,65 @@
...
+i=0
+function upgradeKernel()
+{
+	i=$((i+1))
+	regex =$1
+	kernelver=`uname -r`
+	if [[ "$kernelver" =~ $regex ]]; 
+	then
+	        cd "update${i}"
+        	kmodrpm=`ls kmod-microsoft-hyper-v-*.x86_64.rpm`
```
Comment (prashantmhetar) [inline LISISO/CentOS73/upgrade.sh]: same as above, can we make common file which can be sourced?

File: LISISO/CentOS73/install.sh (hunk 2 above)
Comment (DivyaJyothiKasam) [reply LISISO/CentOS73/install.sh]: done

**Comment by DivyaJyothiKasam** [comment]
Created tar file:
lis-rpm05-2018-12-05.13.10.tar.gz

Below are the logs:
[root@centos74x64 ~]# yum list installed | grep micro
Repodata is over 2 weeks old. Install yum-cron? Or run: yum makecache fast
kmod-microsoft-hyper-v.x86_64 rpm05-20181205 installed
microcode_ctl.x86_64 2:2.1-22.5.el7_4 @updates
microsoft-hyper-v.x86_64 rpm05-20181205 installed
microsoft-hyper-v-debuginfo.x86_64 rpm05-20181205 installed

[root@centos74x64 ~]# modinfo hv_vmbus
filename: /lib/modules/3.10.0-862.6.3.el7.x86_64/weak-updates/microsoft-hyper-v/hv_vmbus.ko
version: rpm05
license: GPL
retpoline: Y
rhelversion: 7.5
srcversion: 581BFCB32D09E7A1B019591
alias: acpi*:VMBus:*
alias: acpi*:VMBUS:*
depends:
vermagic: 3.10.0-862.el7.x86_64 SMP mod_unload modversions
parm: affinity_mode:vmbus channel cpu affinity mode: 0, 1 (int)

**Comment by mboitei** [comment]
Have you tested with ant LIS build ? Please copy the build logs.

**Comment by mboitei** [comment]
Prashant Please approve, so that I can merge

**Review by prashantmhetar — APPROVED** [review]

**Pull request merged**